frontend/src/components/assessment-creation/DisplayAssessmentsPage.tsx

```diff
@@ -0,0 +1,287 @@
+          assessment.region.toLowerCase().includes(search.toLowerCase()) ||
+          assessment.status.toLowerCase().includes(search.toLowerCase()) ||
+          assessment.type.toLowerCase().includes(search.toLowerCase()),
+      );
+    }
+    return filteredTests?.map(getAssessments);
+  }, [search]);
+
+  const assessments = React.useMemo(() => {
+    let sortedAssessments: AssessmentType[] = filteredAssessements as AssessmentType[];
+    if (sortOrder === "descending") {
+      sortedAssessments = sortedAssessments?.sort((a, b) =>
+        a[sortProperty as keyof AssessmentType].toLowerCase() <
+        b[sortProperty as keyof AssessmentType].toLowerCase()
+          ? 1
+          : -1,
+      );
+    } else if (sortOrder === "ascending") {
+      sortedAssessments = sortedAssessments?.sort((a, b) =>
+        a[sortProperty as keyof AssessmentType].toLowerCase() >
+        b[sortProperty as keyof AssessmentType].toLowerCase()
+          ? 1
+          : -1,
+      );
+    }
+    return sortedAssessments;
+  }, [filteredAssessements, sortProperty, sortOrder]);
+
+  return (
+    <>
+      <Box>
+        <HStack justifyContent="space-between">
+          <Text
+            textStyle="header4"
+            color="blue.300"
+            style={{ textAlign: "left" }}
+            marginBottom="0.5em"
+          >
+            Assessments
+          </Text>
+          <CreateAssessementModel />
+        </HStack>
+      </Box>
+      {/* {loading && (
+        <Center margin="15%" flex="1">
+          <LoadingState />
+        </Center>
+      )}
+      {error && (
+        <Center margin="15%" flex="1">
+          <ErrorState />
+        </Center>
+      )} */}
+      {/* {data3 && !error && !loading && ( */}
+      <Box flex="1">
+        <Tabs marginTop={3}>
+          <TabList>
+            <Tab onClick={() => setSearch("")} color={unselectedColor}>
+              All
+            </Tab>
+            <Tab onClick={() => setSearch("Draft")} color={unselectedColor}>
+              Drafts
+            </Tab>
+            <Tab onClick={() => setSearch("Published")} color={unselectedColor}>
+              Published
+            </Tab>
+            <Tab onClick={() => setSearch("Archived")} color={unselectedColor}>
+              Archived
+            </Tab>
+          </TabList>
+          <TabPanels>
+            <TabPanel>
+              <VStack pt={4} spacing={6}>
+                <HStack width="100%">
+                  <InputGroup width="95%">
+                    <Input
+                      borderRadius="6px"
+                      borderColor="grey.100"
+                      backgroundColor="grey.100"
+                      onChange={(e) => setSearch(e.target.value)}
+                      placeholder="Search bar"
+                    />
+                    <InputRightElement pointerEvents="none" h="full">
+                      <SearchOutlineIcon />
+                    </InputRightElement>
+                  </InputGroup>
```

Comment:
Can we use the existing `SearchBar` component here? Or is there different logic being implemented here?